### PR TITLE
org.quetoo.Quetoo

### PR DESCRIPTION
**Pull request by maci0**

### CODE REVIEW

**Comment by nedrichards** [comment]
This needs to be updated to point to git tags or commits representing releases. [Flathub requires stable, reproducible builds](https://github.com/flathub/flathub/wiki/Review-Guidelines).

**Comment by nedrichards** [comment]
Could you add an [OARS description](https://odrs.gnome.org/oars) to the appdata? It should just be a matter of checking the boxes and copying and pasting the generated XML. It'll help make the app info page in the users software center much more useful.

**Comment by maci0** [comment]
done,
on a side note, do you see any reason why the gnome-software center would not properly display all the AppData metadata like license, screenshots, homepage etc?  Doesn't show up properly on my local test install.

**Comment by nedrichards** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/1046

**Comment by nedrichards** [comment]
Er, I don't really. How some of that works is a bit black magic to me.

File: org.quetoo.Quetoo.app [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ID=org.quetoo.Quetoo
```
Comment (TingPing) [inline org.quetoo.Quetoo.app]: This file isn't used by anything?

**Comment by maci0** [comment]
all of the example .json file in the gnome git had an .app file like this, so I thought it may be needed somewhere.

**Comment by TingPing** [comment]
Nah that was just for GNOME tooling, its not used by flatpak or flathub.

**Comment by maci0** [comment]
Ok, removed the file.

**Comment by nedrichards** [comment]
bot, build org.quetoo.Quetoo

**Comment by maci0** [comment]
@nedrichards If I were to remove the branch name and only reference the gitref in the build config would that work ?

**Comment by maci0** [comment]
Ok, fixed the build @nedrichards can you trigger the bot again ?

**Comment by nedrichards** [comment]
bot, build org.quetoo.Quetoo

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.quetoo.Quetoo

**Comment by maci0** [comment]
@nedrichards thanks, also I still need write-access to the repo.
And btw the buildbot seems to hang here https://flathub.org/builds/#/builders/2/builds/623

**Comment by TingPing** [comment]
@maci0 An invite was sent you have to accept it.

**Pull request closed without merging**